runtime/src/lib.rs

```diff
@@ -1907,6 +1907,8 @@ impl order_book::Config for Runtime {
     const MIN_ORDER_LIFESPAN: Moment = MILLISECS_PER_BLOCK; // TODO: order-book clarify
     const MILLISECS_PER_BLOCK: Moment = MILLISECS_PER_BLOCK;
     const MAX_PRICE_SHIFT: Perbill = Perbill::from_percent(50); // TODO: order-book clarify
+    const SMALL_MIN_MAX_RATIO: usize = 1000;
+    const BIG_MIN_MAX_RATIO: usize = 4000;
```

Comment:
I'm thinking if instead of SMALL/BIG naming it SOFT/HARD would make more sense 🤔  (as in soft/hard limit)